hazelcast-documentation/src/HazelcastSimulator/ExecutingASimulatorTest.md

```diff
@@ -1,185 +1,310 @@
+INFO  09:58:26 Waiting for localWarmup completion: 00d 00h 00m 41s
+INFO  09:58:32 Completed Test local warmup
+INFO  09:58:32 Starting Test global warmup
+INFO  09:58:33 Completed Test global warmup
+INFO  09:58:33 Starting Test start
+INFO  09:58:34 Completed Test start
+INFO  09:58:34 Test will run for 00d 00h 01m 00s
+INFO  09:59:04 Running 00d 00h 00m 30s   50.00% complete
+INFO  09:59:34 Running 00d 00h 01m 00s  100.00% complete
+INFO  09:59:34 Test finished running
+INFO  09:59:34 Starting Test stop
+INFO  09:59:36 Completed Test stop
+INFO  09:59:37 Starting Test global verify
+INFO  09:59:39 Completed Test global verify
+INFO  09:59:39 Starting Test local verify
+INFO  09:59:41 Completed Test local verify
+INFO  09:59:41 Starting Test global tear down
+INFO  09:59:43 Finished Test global tear down
+INFO  09:59:43 Starting Test local tear down
+INFO  09:59:45 Completed Test local tear down
+INFO  09:59:45 Terminating workers
+INFO  09:59:45 All workers have been terminated
+INFO  09:59:45 Starting cool down (10 sec)
+INFO  09:59:55 Finished cool down
+INFO  09:59:55 Total running time: 133 seconds
+INFO  09:59:55 -----------------------------------------------------------------------------
+INFO  09:59:55 No failures have been detected!
+INFO  09:59:55 -----------------------------------------------------------------------------```
+
+
+* Now you need logs and results that produced by workers. You can get these requirements from agents via `provisioner`.  
+
+	``` provisioner --download ```
+
+This line performs the following.
 
-First, you should add the simulator worker directory path to bash as an enviroment variable:
 ```
-export SIMULATOR_HOME=~/hazelcast-simulator-0.5
-PATH=$SIMULATOR_HOME/bin:$PATH
+INFO  10:05:41 Hazelcast Simulator Provisioner
+INFO  10:05:41 Version: 0.5, Commit: c6e82c5, Build Time: 02.07.2015 @ 09:50:21 UTC
+INFO  10:05:41 SIMULATOR_HOME: /disk1/hasan/hazelcast-simulator-0.5
+INFO  10:05:41 Loading simulator.properties: /disk1/hasan/exampleSandbox/simulator.properties
+INFO  10:05:42 ==============================================================
+INFO  10:05:42 Download artifacts of 4 machines
+INFO  10:05:42 ==============================================================
+INFO  10:05:42 Downloading from 54.211.146.186
+INFO  10:05:42 Downloading from 54.166.1.79
+INFO  10:05:42 Downloading from 54.147.196.63
+INFO  10:05:42 Downloading from 54.144.235.111
+INFO  10:05:43 ==============================================================
+INFO  10:05:43 Finished Downloading Artifacts of 4 machines
+INFO  10:05:43 ==============================================================
+INFO  10:05:43 Shutting down Provisioner...
+INFO  10:05:43 Done!
 ```
+ 
+After it is completed, the artifacts (log files) are downloaded into the `workers` folder in the working directory.
+
+
+* If want to terminate the instances, just run this line of command.
+
+	```provisioner --terminate```
```

Comment:
Add a line of comment that if an ec2 machine with an agent running is not used for 2 hours, the machines will terminate themselves. To prevent running into a big bill from ec2.